traffic_ops/ort/traffic_ops_ort.pl

```diff
@@ -1847,7 +1855,12 @@ sub get_cfg_file_list {
 	my $cdn_name;
 	my $uri = "/api/1.4/servers/$host_name/configfiles/ats";
 
+	# Telling atstccfg to use-cache=false will cause it to delete the cache directory
+	# Which is what we want: when ORT starts to run, delete the cache. We only want to use the atstccfg cache within the same ORT run, not across different runs.
+	my $real_use_cache = $use_cache;
+	$use_cache = 0;
```

Comment:
This is a very sad way to pass an argument, even for perl. :(

Reply:
It's following the idiom of other arguments 🤷